aws_list_all/generate_html.py

```diff
@@ -0,0 +1,225 @@
+    print('.nfound {border: 10px solid Gold; border-radius: 10px; padding: 10px;}\n')
+    print('.found {border: 10px solid LimeGreen; border-radius: 10px; padding: 10px;}\n')
+    print('.error {border: 10px solid Red; border-radius: 10px; padding: 10px;}\n')
+    print('.denied {border: 10px solid DarkOrange; border-radius: 10px; padding: 10px;}\n')
+    print('.nCollapse {background-color: Gold; border-radius: 10px; color: white; cursor: pointer; '
+        + 'padding: 14px; width: 450px; border: none; text-align: center; font-size: 20px;}\n')
+    print('.fCollapse {background-color: LimeGreen; border-radius: 10px; color: white; cursor: pointer; '
+        + 'padding: 14px; width: 450px; border: none; text-align: center; font-size: 20px;}\n')
+    print('.eCollapse {background-color: Red; border-radius: 10px; color: white; cursor: pointer; '
+        + 'padding: 14px; width: 450px; border: none; text-align: center; font-size: 20px;}\n')
+    print('.dCollapse {background-color: DarkOrange; border-radius: 10px; color: white; cursor: pointer; '
+        + 'padding: 14px; width: 450px; border: none; text-align: center; font-size: 20px;}\n')
+    print('.active, .nCollapse:hover {width: 450px; background-color: #777;}\n')
+    print('.active, .fCollapse:hover {width: 450px; background-color: #777;}\n')
+    print('.active, .eCollapse:hover {width: 450px; background-color: #777;}\n')
+    print('.active, .dCollapse:hover {width: 450px; background-color: #777;}\n')
+    print('.content {display: none; overflow: hidden; width: 450px; background-color: #f1f1f1;}\n')
+
+    print(""".popup {position: relative; display: inline-block; cursor: pointer; 
+        -webkit-user-select: none; -ms-user-select: none; user-select: none;}\n""")
+    print(""".popup .popuptext {visibility: hidden; width: 160px; background-color: #555; color: #fff; 
+        text-align: center; border-radius: 6px; padding: 8px 0; position: absolute; z-index: 1; 
+        bottom: 125%; left: 50%; margin-left: -80px;}\n""")
+    print(""".popup .popuptext::after {content: ""; position: absolute; top: 100%; left: 50%; margin-left: -5px; 
+        border-width: 5px; border-style: solid; border-color: #555 transparent transparent transparent;}\n""")
+    print('.popup .show {visibility: visible; -webkit-animation: fadeIn 1s; animation: fadeIn 1s;}\n')
+    print('@-webkit-keyframes fadeIn {from {opacity: 0;} to {opacity: 1;}}\n')
+    print('@keyframes fadeIn {from {opacity: 0;} to {opacity:1 ;}}\n')
+
+    print('#searchInput {\n')
+    print('  background-position: 10px 10px;\n')
+    print('  background-repeat: no-repeat;\n')
+    print('  width: 300px;\n')
+    print('  font-size: 16px;\n')
+    print('  padding: 12px 20px 12px 40px;\n')
+    print('  border: 1px solid #ddd;\n')
+    print('  margin-bottom: 12px;\n')
+    print('}\n')
+    print('</style>\n')
+    print('</head>\n')
```

Comment:
The static stuff should probably go into a CONSTANT with triple quotes """, then it can be multiline?